docs/source/index.rst

```diff
@@ -103,6 +103,7 @@ Documentation
    reference/quota
    reference/logging
    reference/faq
+   reference/cloud-permissions
```

Comment:
Thinking we should probably add a new section with one of these names:
- Cloud Administration
- Cloud Administration and Usage
- Cloud Setup and Usage
- better name

?

And then we can put
- cloud-permissions: Minimal Cloud Permissions Setup
- cloud-administration: Using Cloud-specific Features
  - may want to rename the filename too 
- quota: Requesting Quota

under it. Note I put some renamed titles above. How do they read?

Reply:
Good point. I also re-ordered the "User Guides" a bit to de-emphasize the local mode. I also chose to use shorter name to make sure they can display in one line in the content table.